content/docs/examples/userauthentication.md

```diff
@@ -78,47 +107,69 @@ We need one more value from Auth0: **the `issuer_url` for your Auth0 account**.
 {{<break 2>}}
 
 ### Creating our Authorizer
-We can now create an `Authorizer` for our Cdev API. 
+We can now create an `Authorizer` for our Cdev API. An `Authorizer` is a function that adds an additional layer of access control to an API endpoint. For example, preventing an authenticated users from accessing data that requires admin authorization.
```

Comment:
I think the second sentence can be clarified to something like: 
The `Authorizer` provides the information needed to configure a JWT authorizer that adds an additional layer of access control to an API endpoint.